pkg/kubernetes/pod/sidecars.go

```diff
@@ -0,0 +1,59 @@
+package pod
+
+import (
+	"fmt"
+	"github.com/openservicemesh/osm-health/pkg/kuberneteshelper"
+	"github.com/openservicemesh/osm-health/pkg/logger"
```

Comment:
These two could go in the last section.  All `openservicemesh/osm-health` should be in the last section of imports, separated with a new line from the rest of the imports.